test/k8sdeployment/deployment_test.go

```diff
@@ -161,9 +163,9 @@ func TestKubernetesDeployment(t *testing.T) {
 		t.Run(fmt.Sprintf("%s %s", tc.Qname, dns.TypeToString[tc.Qtype]), func(t *testing.T) {
 			res, err := kubernetes.DoIntegrationTest(tc, namespace)
 			if err != nil {
-				t.Errorf(err.Error())
+				t.Error(err)
 			}
-			test.CNAMEOrder(t, res)
+			test.CNAMEOrder(res)
```

Comment:
Need to check result, and `t.Error()` if `test.CNAMEOrder(res)` returns an error.